doc/nvim-tree-lua.txt

```diff
@@ -424,6 +425,10 @@ Window / buffer setup.
     Only works when |nvim-tree.view.side| is `left` or `right`.
       Type: `boolean`, Default: `false`
 
+    *nvim-tree.view.centralize_selection*
+    Reposition the view so that the current node is initially centralized, see |zz|.
```

Comment:
Please clarify that this is on BufEnter.

Perhaps: "When entering nvim-tree, reposition the view so that the current node is moved to the center, see |zz|.